data/utils/getCountryCodes.js

```diff
@@ -0,0 +1,10 @@
+const getCountryCodes = (countries, country) => {
+    return countries.find(findCountry => {
```

Comment:
I had a bit of a hard time understanding what `findcountry` was in this context when I first read it, and what the parameter was expected to look like.

Would you mind adding either a comment, and/or a slightly more descriptive variable name here?